### ISSUE TITLE
[Bug] Error parsing json from real-debrid

### ISSUE THREAD

**Issue by loupr** [itsToggle/plex_debrid#112]
When running the script with real-debrid as a debrid provider, there seem to be errors when parsing json data received from real-debrid. Example of the log is shown below:
`
plex_debrid    | [2022-09-11 18:53:18.504656] [realdebrid] error: b''
plex_debrid    | [2022-09-11 18:53:18.504847] [realdebrid] error: (json exception): Expecting value: line 1 column 1 (char 0)
plex_debrid    | [2022-09-11 18:53:18.513353] [realdebrid] error: b'{\n\t"id": "REDACTEDID",\n\t"uri": "https:\\/\\/api.real-debrid.com\\/rest\\/1.0\\/torrents\\/info\\/REDACTEDID"\n}'
`

Even though this error occurs, the cached releases are still added and visible in the webdav mount. I'm unsure as to what is causing this issue, as I've experienced it before on a different host. Maybe it has something to do with the python version used by the docker container?

My setup is running plex, rclone_rd, plex_debrid, prowlarr and a vpn client in docker, while all services are behind the vpn except plex, although I don't believe this is causing the issue. Docker is running on Ubuntu Linux 20.04.1 LTS.

**Comment by itsToggle** [comment]
Hey, do you have the setting "debug printing" enabled?
Usually this error occurs in a safe way and should not be printed to the UI.
The error usually occurs when plex_debrid is trying to add a release to realdebrid. To do this, plex_debrid will first make a "add magnet" call to the realdebrid api and as soon as the request is complete, plex_debrid will make a "torrents/info" call. If the torrent has some sort of problem (e.g. removed by realdebrid, dead torrent, etc) the info response will not be in json format and the exception that you are describing will be caught and the error will be printed as "debug printing". Plex_debrid will delete the broken torrent afterwards.
So tldr; the error is bound to occur sometimes and is nothing to worry about - thought it should only be printed if debug printing is enabled.

**Comment by loupr** [comment]
The debug printing setting was indeed activated and from what I can tell this mainly occurred when the answer from real debris contained a 204 status code. From what I can tell in the code, the response from the get request to real debrid is logged through logerror and parsed no matter the status code of the response, which seems to throw this error when there is indeed no content available, or another status than 200. It might be beneficial to implement a check for the status of the response before handling it, but as you’ve said it doesn’t effect the functionality so it can also well be ignored.
